rclpy/rclpy/executors.py

```diff
@@ -905,3 +905,21 @@ def spin_once_until_future_complete(
     ) -> None:
         future.add_done_callback(lambda x: self.wake())
         self._spin_once_impl(timeout_sec, future.done)
+
+    def shutdown(
+        self,
+        timeout_sec: float = None,
+        wait_for_threads: bool = True
+    ) -> bool:
+        """
+        Stop executing callbacks and wait for their completion.
+        :param timeout_sec: Seconds to wait. Block forever if ``None`` or negative.
```

Comment:
```suggestion
        Stop executing callbacks and wait for their completion.

        :param timeout_sec: Seconds to wait. Block forever if ``None`` or negative.
```
I think the linter wants this extra whitespace

Reply:
fixed in https://github.com/ros2/rclpy/pull/1309/commits/43ce2d74e8636b523135163ad2dc8f9bf4b1f6f3